### PR TITLE
[libevent-2.1.10] Pin specific versions of some dependencies

### PR DESCRIPTION
**Pull request by G-D-Petrov**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by G-D-Petrov** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -8,4 +8,7 @@ lmdb:
   - "0.9.22"
 folly:
   # This version of folly is used to make the build usable by more environments
-  - "2023.09.25.00"	
+  - "2023.09.25.00"
+libprotobuf:
+  # This version of libprotobuf is used to make the build ubasable by BQNT
+  - "3.21.12"
```
Comment (jjerphan) [inline recipe/conda_build_config.yaml:14]: The re-rendering has not happened, which is slightly weird.

Could you try to pin this version of `libprotobuf` in `recipe/meta.yaml`?

**Comment by G-D-Petrov** [comment]
@conda-forge-admin, please rerender

**Comment by G-D-Petrov** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Review by poodlewars — APPROVED** [review]

**Review by JohanMabille — APPROVED** [review]

**Comment by jjerphan** [comment]
Dependencies' versions are specified in a `index.json` file inside the `info` tarball itself inside the conda package.

<details>
<summary> <a href="https://anaconda.org/conda-forge/arcticdb/4.0.0/download/linux-64/arcticdb-4.0.0-py39h1278ff5_2.conda"><code>linux-64_arcticdb-4.0.0-py39h1278ff5_2</code></a>'s <code>index.json</code></summary>

```json
{
  "arch": "x86_64",
  "build": "py39h1278ff5_2",
  "build_number": 2,
  "depends": [
    "attrs",
    "aws-crt-cpp >=0.19.8",
    "aws-sdk-cpp >=1.10.57,<1.10.58.0a0",
    "azure-core-cpp >=1.10.2,<2.0a0",
    "azure-identity-cpp >=1.5.1,<2.0a0",
    "azure-storage-blobs-cpp >=12.9.0,<13.0a0",
    "dataclasses",
    "decorator",
    "double-conversion >=3.3.0,<3.4.0a0",
    "fmt >=9.1.0,<10.0a0",
    "folly >=2023.9.25.0,<2023.9.25.1.0a0",
    "gflags >=2.2.2,<2.3.0a0",
    "glog >=0.6.0,<0.7.0a0",
    "grpcio-tools",
    "gtest >=1.14.0,<1.14.1.0a0",
    "libevent >=2.1.10,<2.1.11.0a0",
    "libgcc-ng >=12",
    "libmongocxx >=3.7.2,<3.8.0a0",
    "libprotobuf >=3.21.12,<3.22.0a0",
    "libstdcxx-ng >=12",
    "lmdb",
    "lz4-c >=1.9.3,<1.10.0a0",
    "msgpack-c >=6.0.0,<7.0a0",
    "msgpack-python",
    "numpy",
    "packaging",
    "pandas",
    "pcre >=8.45,<9.0a0",
    "prometheus-cpp >=1.1.0,<1.2.0a0",
    "prometheus_client",
    "protobuf",
    "psutil",
    "pymongo",
    "python >=3.9,<3.10.0a0",
    "python-xxhash",
    "python_abi 3.9.* *_cp39",
    "pyyaml",
    "six",
    "spdlog >=1.11.0,<1.12.0a0",
    "xxhash >=0.8.2,<0.8.3.0a0",
    "zstd >=1.5.5,<1.6.0a0"
  ],
  "license": "BUSL-1.1",
  "license_family": "OTHER",
  "name": "arcticdb",
  "platform": "linux",
  "subdir": "linux-64",
  "timestamp": 1695911714514,
  "version": "4.0.0"
}
```

</details>

<details>
<summary> <a href="https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=830156&view=artifacts&pathAsName=false&type=publishedArtifacts">This build of (as of 60ccbe4)</a>'s <code>index.json</code></summary>

```json
{
  "arch": "x86_64",
  "build": "py39hef22b89_4",
  "build_number": 4,
  "depends": [
    "attrs",
    "aws-crt-cpp >=0.19.8",
    "aws-sdk-cpp >=1.10.57,<1.10.58.0a0",
    "azure-core-cpp >=1.10.2,<2.0a0",
    "azure-identity-cpp >=1.5.1,<2.0a0",
    "azure-storage-blobs-cpp >=12.9.0,<13.0a0",
    "dataclasses",
    "decorator",
    "double-conversion >=3.3.0,<3.4.0a0",
    "fmt >=9.1.0,<10.0a0",
    "folly >=2023.9.25.0,<2023.9.25.1.0a0",
    "gflags >=2.2.2,<2.3.0a0",
    "glog >=0.6.0,<0.7.0a0",
    "grpcio-tools",
    "gtest >=1.14.0,<1.14.1.0a0",
    "libevent >=2.1.10,<2.1.11.0a0",
    "libgcc-ng >=12",
    "libmongocxx >=3.7.2,<3.8.0a0",
    "libprotobuf >=3.21.12,<3.22.0a0",
    "libstdcxx-ng >=12",
    "lmdb",
    "lz4-c >=1.9.3,<1.10.0a0",
    "msgpack-c >=6.0.0,<7.0a0",
    "msgpack-python",
    "numpy",
    "packaging",
    "pandas",
    "pcre >=8.45,<9.0a0",
    "prometheus-cpp >=1.1.0,<1.2.0a0",
    "prometheus_client",
    "protobuf",
    "psutil",
    "pymongo",
    "python >=3.9,<3.10.0a0",
    "python-xxhash",
    "python_abi 3.9.* *_cp39",
    "pyyaml",
    "six",
    "spdlog >=1.11.0,<1.12.0a0",
    "xxhash >=0.8.2,<0.8.3.0a0",
    "zstd >=1.5.5,<1.6.0a0"
  ],
  "license": "BUSL-1.1",
  "license_family": "OTHER",
  "name": "arcticdb",
  "platform": "linux",
  "subdir": "linux-64",
  "timestamp": 1700843184605,
  "version": "4.1.0"
}
```

</details>

I think we are safe merging it.

**Review by jamesmunro — APPROVED** [review]

**Pull request merged**